### PR TITLE
Allow user to config xdebug from environment variables

### PR DESCRIPTION
**Pull request by anaelChardan**
I finally arrived to configure this :-)

### CODE REVIEW

File: COMPOSE.md [hunk 1]
```diff
@@ -105,6 +105,11 @@ $ docker-compose exec akeneo-behat pim-initialize
 
 As *xdebug* is activated by default on development images, it is recommended to first deactivate it, then run the initialization, and finally enable it again.
 
+Also, you can configure two things on Xdebug through environment variable on image akeneo and akeneo-behat.
```
Comment (damien-carcel) [inline COMPOSE.md]: through environment variables on akeneo and akeneo-behat images.

Comment (damien-carcel) [inline COMPOSE.md]: These environment variables are all optional.

File: php/Dockerfile [hunk 2]
```diff
@@ -2,6 +2,23 @@ FROM debian:jessie
 MAINTAINER Damien Carcel <damien.carcel@gmail.com>
 
 ENV DEBIAN_FRONTEND noninteractive
+ENV GOSU_VERSION 1.9
+
+# Install user impersonator
+RUN set -x \
```
Comment (damien-carcel) [inline php/Dockerfile]: Do that after the Install block below

File: php/Dockerfile [hunk 3]
```diff
@@ -2,6 +2,23 @@ FROM debian:jessie
...
+    && apt-get update && apt-get install -y --no-install-recommends ca-certificates wget && rm -rf /var/lib/apt/lists/* \
```
Comment (damien-carcel) [inline php/Dockerfile]: Use less, it is installed in the following block.

File: php/Dockerfile [hunk 4]
```diff
@@ -2,6 +2,23 @@ FROM debian:jessie
 MAINTAINER Damien Carcel <damien.carcel@gmail.com>
 
 ENV DEBIAN_FRONTEND noninteractive
+ENV GOSU_VERSION 1.9
```
Comment (damien-carcel) [inline php/Dockerfile]: To remove

File: php/Dockerfile [hunk 5]
```diff
@@ -2,6 +2,23 @@ FROM debian:jessie
...
+
+# Install user impersonator
+RUN set -x \
+    && apt-get update && apt-get install -y --no-install-recommends ca-certificates wget && rm -rf /var/lib/apt/lists/* \
+    && dpkgArch="$(dpkg --print-architecture | awk -F- '{ print $NF }')" \
```
Comment (damien-carcel) [inline php/Dockerfile]: To remove

File: php/Dockerfile [hunk 6]
```diff
@@ -2,6 +2,23 @@ FROM debian:jessie
...
+    && wget -O /usr/local/bin/gosu "https://github.com/tianon/gosu/releases/download/$GOSU_VERSION/gosu-$dpkgArch" \
```
Comment (damien-carcel) [inline php/Dockerfile]: Use directly version and architecture (next line too) rather than env variables.

File: php/Dockerfile [hunk 7]
```diff
@@ -2,6 +2,23 @@ FROM debian:jessie
...
+    && wget -O /usr/local/bin/gosu.asc "https://github.com/tianon/gosu/releases/download/$GOSU_VERSION/gosu-$dpkgArch.asc" \
```
Comment (damien-carcel) [inline php/Dockerfile]: As you need wget, make it like PHP 7 images: install the various tools (wget included), then add the repo for GOSU, then install gosu, then install PHP (each then being a RUN instruction).

File: php/Dockerfile [hunk 8]
```diff
@@ -21,17 +38,18 @@ RUN useradd docker --shell /bin/bash --create-home \
...
-RUN php5enmod mcrypt && \
-    php5enmod xdebug
+RUN php5enmod mcrypt
 
 RUN sed -i "s/;date.timezone =/date.timezone = Europe\/Paris/" /etc/php5/cli/php.ini && \
     sed -i "s/memory_limit = .*/memory_limit = 2G/" /etc/php5/cli/php.ini && \
     sed -i "s/upload_max_filesize = .*/upload_max_filesize = 20M/" /etc/php5/cli/php.ini && \
     sed -i "s/post_max_size = .*/post_max_size = 20M/" /etc/php5/cli/php.ini
 
+COPY ./files/carcel-php-entrypoint.sh /home/docker
```
Comment (damien-carcel) [inline php/Dockerfile]: just `entrypoint.sh` is enought, put it in `/usr/local/bin`.

File: php/Dockerfile [hunk 9]
```diff
@@ -21,17 +38,18 @@ RUN useradd docker --shell /bin/bash --create-home \
...
+RUN chmod a+x /home/docker/carcel-php-entrypoint.sh
+
 # Define "docker" as current user
 USER docker
 WORKDIR /home/docker/
+
+ENTRYPOINT ["/home/docker/carcel-php-entrypoint.sh"]
```
Comment (damien-carcel) [inline php/Dockerfile]: /usr/local/bin/entrypoint.sh

**Review by damien-carcel — CHANGES_REQUESTED** [review]
Some changes and it wil be GTM :).
Thanks for this 👍

File: COMPOSE.md (hunk 1 above)
Comment (anaelChardan) [reply COMPOSE.md]: OK

File: php/Dockerfile (hunk 4 above)
Comment (anaelChardan) [reply php/Dockerfile]: OK

File: php/Dockerfile (hunk 2 above)
Comment (anaelChardan) [reply php/Dockerfile]: OK

File: php/Dockerfile (hunk 3 above)
Comment (anaelChardan) [reply php/Dockerfile]: OK

File: php/Dockerfile (hunk 8 above)
Comment (anaelChardan) [reply php/Dockerfile]: OK

File: php/Dockerfile (hunk 9 above)
Comment (anaelChardan) [reply php/Dockerfile]: OK

**Review by damien-carcel — APPROVED** [review]

**Pull request merged**